src/App.tsx

```diff
@@ -1,24 +1,78 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { FilterStatus, Todo } from './types/Todo';
+import { showError, GetFilteredTodos } from './helpers/helpers';
+import { Error } from './components/Error';
+import { Footer } from './components/Footer';
+import { Todos } from './components/Todos';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+const USER_ID = 10881;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [filter, setFilter] = useState(FilterStatus.ALL);
+  const [error, setError] = useState('');
+  const [tempTodo, setTempTodo] = useState<null | Todo>(null);
+
+  const [loadingTodos, setLoadingTodos] = useState<number[]>([]);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => showError('Failed to get list of todos', setError));
```

Comment:
here you use then + catch, below in your code you use async + wait, I suggest using same approache everywhere

Reply:
i like to use then + catch, and i used it in all cases in my app but i had this comment
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/130560419/f5437255-ddb2-4404-9625-941a96dbfa1b)